bukkit/src/main/java/org/bstats/bukkit/Metrics.java

```diff
@@ -62,14 +62,17 @@ public Metrics(Plugin plugin, int serviceId) {
         boolean logSentData = config.getBoolean("logSentData", false);
         boolean logResponseStatusText = config.getBoolean("logResponseStatusText", false);
 
+        String minecraftVersion2 = Bukkit.getServer().getName();
+        boolean folia = (minecraftVersion2.toUpperCase().contains("FOLIA"));
```

Comment:
This wouldn't catch forks of folia, you should detect if the class `io.papermc.paper.threadedregions.RegionizedServer` exists instead.